docs/config.md

```diff
@@ -228,12 +250,12 @@ To load this adapter, you must include the following in
 the `system/controlDict` configuration file of the case:
 
 ```c++
+libs ("libpreciceAdapterFunctionObject.so");
 functions
 {
     preCICE_Adapter
     {
         type preciceAdapterFunctionObject;
-        libs ("libpreciceAdapterFunctionObject.so");
     }
```

Comment:
Add a comment that specifying the `libs` outside of the `functions` dictionary is important to also use the provided boundary conditions for fluid-fluid coupling.